src/config.cc

```diff
@@ -404,6 +411,30 @@ void Config::initFieldCallback() {
         double cutoff = val / 100;
         return srv->storage_->SetColumnFamilyOption(trimRocksDBPrefix(k), std::to_string(cutoff));
       }},
+      {"rocksdb.level_compaction_dynamic_level_bytes", [this](Server* srv, const std::string &k,
+                                                        const std::string& v)->Status {
+        if (!srv) return Status::OK();
+        if (!RocksDB.level_compaction_dynamic_level_bytes) {
+          return Status(Status::NotOK, errNotSetLevelCompactionDynamicLevelBytes);
+        }
+        std::string level_compaction_dynamic_level_bytes = v == "yes" ? "true" : "false";
+        return srv->storage_->SetColumnFamilyOption(trimRocksDBPrefix(k), level_compaction_dynamic_level_bytes);
```

Comment:
Does this option support dynamically changeable through SetOptions() API？I can not find any infomations here: https://github.com/facebook/rocksdb/blob/main/include/rocksdb/advanced_options.h#L594

Reply:
> Does this option support dynamically changeable through SetOptions() API？I can not find any infomations here: https://github.com/facebook/rocksdb/blob/main/include/rocksdb/advanced_options.h#L594

I haven't found too, I changed this option load logic. Please help me review and check code, thanks.